engine/installation/linux/docker-ee/centos.md

```diff
@@ -34,10 +34,11 @@ The `centos-extras` repository must be enabled. This repository is enabled by
 default, but if you have disabled it, you need to
 [re-enable it](https://wiki.centos.org/AdditionalResources/Repositories){: target="_blank" class="_" }.
 
-In addition, you must use the `devicemapper` storage driver if you use
-Docker EE. On production systems, you must use `direct-lvm` mode, which requires
-one or more dedicated block devices. Fast storage such as solid-state media
-(SSD) is recommended.
+In addition, you must use the `overlay2` or `devicemapper` storage driver if you
+use Docker EE. **The `overlay2` driver requires CentOS 7.4 or higher.** On
```

Comment:
Afaik, unlike RHEL, CentOS uses a rolling release, without offering a way to run older versions, so the only version of CentOS we _can_ support is "CentOS 7" (IOW: the latest 7.x version of CentOS).

Note that we're adding feature-detection in an upcoming update, in which case RHEL/CentOS 7.2 and up should be detected to support overlay2